packages/react-mutation-mapper/src/component/lollipopPlot/DomainTooltip.tsx

```diff
@@ -0,0 +1,71 @@
+import { PfamDomain, PfamDomainRange } from 'genome-nexus-ts-api-client';
+import { computed } from 'mobx';
+import { observer } from 'mobx-react';
+import * as React from 'react';
+
+import { MobxCache } from '../../model/MobxCache';
+
+export interface IDomainTooltipProps {
+    mutationAlignerCache?: MobxCache<string>;
+    range: PfamDomainRange;
+    domain: PfamDomain | undefined;
+    pfamDomainId: string;
+}
+
+@observer
+export default class DomainTooltip extends React.Component<
+    IDomainTooltipProps,
+    {}
+> {
+    @computed
+    get mutationAlignerCacheData() {
+        return this.props.mutationAlignerCache &&
+            this.props.mutationAlignerCache.cache
+            ? this.props.mutationAlignerCache.cache[this.props.pfamDomainId]
```

Comment:
accessing the `cache` field is the only way to trigger rerendering for some reason, using the `get` function doesn't work.